hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -84,86 +61,26 @@
 
   private OzoneManagerRatisUtils() {
   }
-  /**
-   * Create OMClientRequest which encapsulates the OMRequest.
-   * @param omRequest
-   * @return OMClientRequest
-   * @throws IOException
-   */
-  public static OMClientRequest createClientRequest(OMRequest omRequest) {
-    Type cmdType = omRequest.getCmdType();
```

Comment:
Can we keep OzoneManagerRatisUtils#createClientRequest unchanged and invoke omVersionFactory#getClientRequest Here?
Now OzoneManagerRatisUtils defines the request/response functions, I prefer to keep this function there. This is just my personal taste for this change.